src/js/view/preview.js

```diff
@@ -18,11 +26,21 @@ define([
         },
         loadImage: function(model, playlistItem) {
```

Comment:
loadImage, is no longer a function which loads images :)

To keep this code tight, we should separate out concerns so that each function has only one responsibility.

1. onPlaylistItem - does this item have an image? Should we load it?
1. loadImage - loads an image

The confusing part is that autostart **does need to load an image, it just doesn't want to display it on startup.